lib/routes/password.js

```diff
@@ -348,6 +351,11 @@ module.exports = function (
               )
               .then(
                 function() {
+                  return request.emitMetricsEvent('password.forgot.send_code')
```

Comment:
I think, for maximum value to the funnel, we should emit other events in addition to the ones you have here.

Imho, for each endpoint we want at least two success events (I plan to do this on other endpoints too). First, we want an event as soon as the handler is invoked, signifying that the user has made a request. Then we want the event you have here, signifying that the request has been completed successfully on the back-end.

We also want to make sure we emit events for all of the different error conditions, although not necessarily in this PR. I have a plan to come back and add those everywhere in a single sweep, once we're happy that the various success events are all showing up correctly.